InsiteTeamTask/ClientApp/src/app/attendance/attendance.component.html

```diff
@@ -0,0 +1,82 @@
+<p>Select Attendence by Game and Season</p>
+
+<div class="row">
+  <div class="col-6">
+    <div class="row">
+      <div class="col-6">Season</div>
+      <div class="col-6">
+        <div *ngIf="seasons">
+          <select (change)="setSeason($event.target.value)">
+            <option disabled="disabled">Select Season</option>
+            <option *ngFor="let season of seasons" value={{season.id}}>{{season.id}}</option>
+          </select>
+        </div>
+      </div>
+    </div>
+    <div class="col-6">
+      <div class="row">
+        <div class="col-6">Game</div>
+        <div class="col-6">
+          <div *ngIf="games">
+            <select (change)="setGame($event.target.value)">
+              <option disabled="disabled">Select Game</option>
+              <option *ngFor="let game of games" value={{game.id}}>{{game.id}}</option>
```

Comment:
Since you have access to `Game` objects, this would be slightly more user-friendly if it was:
```
<option *ngFor="let game of games" [value]="game.id">{{game.description}}</option>
```